tests/integration/test_data/gomod_e2e_multiple_modules/.build-config.json

```diff
@@ -18,8 +18,8 @@
     {
       "kind": "literal",
       "name": "GOSUMDB",
-      "value": "off"
+      "value": "sum.golang.org"
     }
   ],
   "project_files": []
-}
+}
```

Comment:
pro tip: run `make generate-test-data` whenever you need to update integration test data

(it would also fix the missing newlines in this case)